### PR TITLE
feature(condo): Doma-8542 register billing receipt file mutation

### PR DESCRIPTION
**Pull request by dkoviazin**
We need to minimize interaction with external billing integrations in order to save the PDF receipt to CONDO.
We also need the ability to save the file not as multi-part form data, but from a Base64 encoded binary stream.

### CODE REVIEW

File: apps/condo/lang/ru/ru.json [hunk 1]
```diff
@@ -2196,6 +2196,8 @@
   "api.billing.sendNewReceiptMessagesToResidentScopes.INVALID_BILLING_CATEGORY_PROVIDED": "Необходимо передать id существующей биллинговой категории",
   "api.billing.sendNewReceiptMessagesToResidentScopes.SCOPES_IS_EMPTY": "Необходимо передать хотя бы один элемент в scopes",
   "api.billing.sendNewReceiptMessagesToResidentScopes.INVALID_BILLING_PROPERTY_PROVIDED": "Все переданные id домов (billingProperty) в скоупах должны принадлежать организации из переданного биллингового контекста",
+  "api.billing.registerBillingReceiptFile.MULTIPLE_BILLING_RECEIPTS_FOUND": "Найдено несколько квитанций для PDF файла",
```
Comment (toplenboren) [inline apps/condo/lang/ru/ru.json:2199]: I'd rather: found more then 1 billing receipt, as it is more specific

File: apps/condo/lang/en/en.json [hunk 2]
```diff
@@ -2196,6 +2196,8 @@
   "api.billing.sendNewReceiptMessagesToResidentScopes.INVALID_BILLING_CATEGORY_PROVIDED": "Please provide existing non-deleted billing category id",
   "api.billing.sendNewReceiptMessagesToResidentScopes.SCOPES_IS_EMPTY": "Scopes could not be empty",
   "api.billing.sendNewReceiptMessagesToResidentScopes.INVALID_BILLING_PROPERTY_PROVIDED": "All provided billing properties in scopes should be non-deleted & connected to organization of the billing context",
+  "api.billing.registerBillingReceiptFile.MULTIPLE_BILLING_RECEIPTS_FOUND": "There is more then one receipt for this receipt file",
+  "api.billing.registerBillingReceiptFile.NO_RECEIPTS_FOUND": "There is no receipt for this receipt file",
```
Comment (toplenboren) [inline apps/condo/lang/en/en.json:2200]: better to specifically describe which model is not found. BillingReceipt not found

File: apps/condo/domains/billing/constants.js [hunk 3]
```diff
@@ -5,7 +5,20 @@
...
+    REGISTER_BILLING_RECEIPT_FILE_SKIPPED_STATUS,
+]
+
 module.exports = {
     BILLING_VALIDATE_QR_CODE_WINDOW,
     MAX_CLIENT_VALIDATE_QR_CODE_BY_WINDOW,
+    REGISTER_BILLING_RECEIPT_FILE_STATUSES,
+    REGISTER_BILLING_RECEIPT_FILE_CREATED_STATUS,
+    REGISTER_BILLING_RECEIPT_FILE_UPDATED_STATUS,
+    REGISTER_BILLING_RECEIPT_FILE_SKIPPED_STATUS,
```
Comment (toplenboren) [inline apps/condo/domains/billing/constants.js:23]: why export all and not export only REGISTER_BILLING_RECEIPT_FILE_STATUSES

**Review by toplenboren — APPROVED** [review]

File: apps/condo/domains/billing/constants.js (hunk 3 above)
Comment (dkoviazin) [reply apps/condo/domains/billing/constants.js:23]: I need them in tests

**Review by ekabardinsky — APPROVED** [review]

File: apps/condo/lang/ru/ru.json (hunk 1 above)
Comment (dkoviazin) [reply apps/condo/lang/ru/ru.json:2199]: Done

File: apps/condo/lang/en/en.json (hunk 2 above)
Comment (dkoviazin) [reply apps/condo/lang/en/en.json]: Donw

File: apps/condo/domains/billing/access/RegisterBillingReceiptFileService.js [hunk 4]
```diff
@@ -0,0 +1,29 @@
+const { throwAuthenticationError } = require('@open-condo/keystone/apolloErrorFormatter')
+const { getById } = require('@open-condo/keystone/schema')
+
+const { checkBillingIntegrationsAccessRights } = require('@condo/domains/billing/utils/accessSchema')
+const { SERVICE } = require('@condo/domains/user/constants/common')
+
+async function canRegisterBillingReceiptFile ({ authentication: { item: user }, args: { data: { context: { id: contextId } } } } ) {
```
Comment (pahaz) [inline apps/condo/domains/billing/access/RegisterBillingReceiptFileService.js:7]: it looks like runtime error

**Pull request merged**